src/cool_seq_tool/mappers/exon_genomic_coords.py

```diff
@@ -147,37 +301,33 @@ async def tx_segment_to_genomic(
         :param exon_end_offset: Ending exon offset
         :return: GRCh38 genomic data (inter-residue coordinates)
         """
-        resp = GenomicDataResponse(
-            genomic_data=None, warnings=[], service_meta=service_meta()
-        )
-
         # Ensure valid inputs
-        warnings = []
+        errors = []
         if not transcript:
-            warnings.append("Must provide `transcript`")
+            errors.append("Must provide `transcript`")
```

Comment:
I also think this `if` check is unnecessary? IMO if we declare types, we can assume they're followed.

Reply:
Agreed!